src/wp-admin/about.php

```diff
@@ -334,9 +334,9 @@
 				?>
 			</h2>
 			<p><?php printf(
-				__( 'WordPress is working on a new way to create and control your content and we&#8217;d love to have your help. Interested in being an <a href="%s">early tester</a> or getting involved with the Gutenberg project? <a href="%s">Contribute on GitHub</a>.' ),
+				__( 'ClassicPress is working on a new way to create and control your content and we&#8217;d love to have your help. Interested in being an <a href="%s">early tester</a> or getting involved with the Gutenberg project? <a href="%s">Contribute on GitHub</a>.' ),
```

Comment:
:thinking:

Reply:
I was surprised that that was still there, and I see you have addressed it with #28